### ISSUE TITLE
Play services Crashing

### ISSUE THREAD

**Issue by mananwason** [CyanogenMod/android_device_motorola_titan#6]
Google play services is crashing making the version unusable

**Comment by asdwfwqd** [comment]
Same things happened to me. Gapps worked good on my Nexus 9 flounder, but crashed on my XT1063. You can try turn off location services, because if I wipe all partitions and reinstall without gapps, location services still don't working, and Yahoo Weather can not find my current location.

**Comment by monojp** [comment]
Opengapps pico is working fine on my XT1068. BTW there are even ways to use
network location without gapps installed if you prefer that. Search for
"NLP" or "nlpbackend" in the f-droid store.

**Comment by asdwfwqd** [comment]
Yes! I tried the latest opengapps mini, and it worked,thanks for fix it.
